pkg/registrar/map_functions.go

```diff
@@ -25,13 +27,16 @@ import (
 	"github.com/vmware-tanzu/cartographer/pkg/apis/v1alpha1"
 )
 
+//counterfeiter:generate sigs.k8s.io/controller-runtime/pkg/client.Client
+
 //counterfeiter:generate . Logger
 type Logger interface {
 	Error(err error, msg string, keysAndValues ...interface{})
 }
 
 type Mapper struct {
 	Client client.Client
+	// fixme We should accept the context, not the logger - then we get the right logger and so does the client
```

Comment:
makes me wonder if we should update our linter setting to remind us about fixmes/todos 🤔 

https://github.com/matoous/godox

Reply:
No thanks. this needs to be in a metric style static analysis tool such as Sonarqube. These are placeholders that needto be adressed. I leave markers so that if I'm passing by a fixme/todo so that anyone apssing them while making changes understands there is intent to change. Imagine you think to yourself, while working in this code "this really should be a context" - this fixme vailidates your thoughts and saves you searching for a story. 

if we get a good static analysis tool running, we'll be able to decide when to add stories as needed, and we can annotate fixme/todo with story links.

Reply:
makes sense! 👍